hail/python/test/hail/expr/test_types.py

```diff
@@ -82,3 +82,5 @@ def test_coercers_can_coerce(self):
             c = coercer_from_dtype(t)
             self.assertTrue(c.can_coerce(t))
             self.assertFalse(c.requires_conversion(t))
+
+
```

Comment:
I noticed that you removed a test. I would like to see a regression test for this.

Reply:
I didn't remove it, actually. I had added a test to try to reproduce the bug, realized it didn't actually reproduce the bug, and took it back out. I'll remove the whitespace.

Reply:
ah, I see. I thought you'd meant "removed an existing test from the test suite", but you probably just meant "removed the test that I'd added in a previous commit". But yeah, I'm pretty sure the regression test already exists in `test_array_methods`, and the test I'd added turned out to not actually reproduce the bug.

Reply:
Alright. Good to know.